backend/ts/src/main/scala/aqua/backend/ts/TypeScriptService.scala

```diff
@@ -1,20 +1,108 @@
 package aqua.backend.ts
 
+import aqua.backend.air.FuncAirGen
+import aqua.model.transform.res.FuncRes
+import aqua.types.*
+import cats.syntax.show.*
 import aqua.model.transform.res.ServiceRes
 
 case class TypeScriptService(srv: ServiceRes) {
 
-  import TypeScriptFunc.typeToTs
+  import TypeScriptCommon._
+
+  def fnHandler(arrow: ArrowType, memberName: String) = {
+    s"""
+       | if (req.fnName === '${memberName}') {
+       |     ${callBackExprBody(arrow, "service." + memberName)}
+       | }
+    """.stripMargin
+  }
 
   def generate: String =
+    val fnHandlers = srv.members
+      .map({ case (name, arrow) =>
+        fnHandler(arrow, name)
+      })
+      .mkString("\n\n")
+
+    val fnDefs = srv.members
+      .map({ case (name, arrow) =>
+        s"${name}: ${fnDef(arrow)};"
+      })
+      .mkString("\n")
+
+    val registerName = s"register${srv.name}"
+
+    val peerDecl = "peer: FluencePeer";
+    val serviceIdDecl = "serviceId: string";
+    val serviceDecl = s"""service: {
+          ${fnDefs}
+      }"""
+    
+    val registerServiceArgsSource = srv.defaultId.fold(
```

Comment:
i think it will be more clearer if there will be a comment and all strings will be inserted directly into string interpolation